components/entity/EntityDetails.vue

```diff
@@ -56,6 +56,10 @@
         type: Object,
         required: true
       },
+      query: {
+        type: String,
+        default: null
+      },
```

Comment:
Is this used in this component?

Reply:
no, left over from previous iteration when the query was shown alongside the entity title instead of above the results as  is the case now.